### PR TITLE
Instance name in title

### LINKED ISSUE #1463: use instance name in title and views?
@SchrodingersGat this is just a suggestion - close this if you disagree, no hard feelings.

There are a few views and (meta) titles where 'InvenTree' is used where it could make sense to use the instance name. I currently have a few instances deployed in places where English is not widely used or known. A few users, especially in manufacturing, are confused by the 'InvenTree' as they are used to only use the internal system names (for exampel 'TeilDB').

I would make a PR replacing 'InvenTree' with the instance name in all end-user facing parts. About, version and setting-views should keep 'InvenTree' in my opinion as to show what system is used.

### PR DESCRIPTION
**Pull request by matmair**
closes #1463 

- adds a setting for choosing if 'InvenTree' or the instance name should be used in views and title -> defaults to 'InvenTree'
- adds a function to easily query the for that string
- adds a template tag for easy use in templates
- implements new function in current views

### CODE REVIEW

**Comment by matmair** [comment]
@SchrodingersGat are there dev-docs for template tags / templates? I didn't find any so there is no docs for the new tag at this point

File: InvenTree/common/models.py [hunk 1]
```diff
@@ -58,6 +58,13 @@ class InvenTreeSetting(models.Model):
             'description': _('String descriptor for the server instance'),
         },
 
+        'INVENTREE_INSTANCE_TITLE': {
+            'name': _('Use Instance Name'),
+            'description': _('Use the instance name in the Titel-Bar'),
```
Comment (SchrodingersGat) [inline InvenTree/common/models.py:63]: "Titel" should be "Title"

**Comment by SchrodingersGat** [comment]
Nice work @matmair a very clean implementation.

> are there dev-docs for template tags / templates? I didn't find any so there is no docs for the new tag at this point

Currently there are no separate developer docs...

File: InvenTree/common/models.py (hunk 1 above)
Comment (matmair) [reply InvenTree/common/models.py:63]: oh sorry, my german is showing. Will fix ASAP

**Comment by matmair** [comment]
> 
> 
> Nice work @matmair a very clean implementation.
>
I love Django for how easy KISS is with it :-)
Thank you, I am trying to use all the interfaces you laid out @SchrodingersGat

**Comment by SchrodingersGat** [comment]
Nice work, thanks

**Pull request merged**